### ISSUE TITLE
Transfer to Macchina organization?

### ISSUE THREAD

**Issue by adamvoss** [macchina/LIN#4]
Hi Collin,

I'm working on reorganizing Macchina's libraries on GitHub.  This will include splitting the LIN library to its own repository.  I notice you have already done this.  Since users have filled issues against your repository I am wondering if you would allow this repository to be transferred to the [macchina organization](https://github.com/macchina)?  Once that is done, you would be able to fork it to have your own copy again.

Thanks!
Adam

/cc @rocketjosh

**Comment by collin80** [comment]
Sure, I only brought it out to a separate repo because that seemed like the right thing to do. I think I did mention to Josh that I thought that Macchina should really do that same thing and then I could just fork those repos and work on things myself. Anyway, I'm willing to do it but so far as I know you can only transfer a repo to an organization you are an admin of. Well, I'm not an admin of the Macchina organization so I can't do this.

**Comment by collin80** [comment]
Sure enough, just tried and it says I don't have proper permissions to create a repo on Macchina. You'd have to have someone add me for such permissions. Then I can transfer any of the libraries you want. I might have separated out some others as well but maybe they don't have any issues attached and thus you could just recreate them. Let me know.

**Comment by collin80** [comment]
Done!

**Comment by adamvoss** [comment]
Thanks Collin!  As a workaround to the membership issue, the repositories could have been transferred to me.  But before I could ask if you wanted to me a member of the org so I could check to make sure it was okay, I was told to just do it :-).

As far as which to transfer, I would say definitely this one.  For the others, there are two issues.  The first is that I am still tracking down the source for some of them to know whether they should be a fork to begin with.

The second is they tend to have your modifications.  I _assume_ these would all be accepted upstream so they can be transfered, but don't know who decides that at this point.  If you and @rocketjosh and weigh in on the 2nd point about upstreaming your changes, I will finish trying to track down origin repositories.

**Comment by collin80** [comment]
Well, if I changed something then chances are the changes are required in order to compile and use M2RET. So, they could be important. Still, ultimately Macchina should be able to review and potentially reject changes. So, ideally my changes should really start out in my repo version and then transition to the Macchina repo through pull requests. That way I can do whatever I want and Macchina has the final say of what ends up in the official "Macchina" version. But, it's up to them whether they grab everything as-is and we start from there. As for tracking things down, absolutely you should find the original source and, if possible, Macchina should fork from the original and then apply any changes we've done on top. That way you can track upstream changes from the original source. Also, that helps to credit the original source of the code as well. So, probably the best thing to do is to go fork from the original git repos where possible and then see what was changed for the versions I've got and re-apply them on top. Then I'll issue pull requests if I change anything else.

**Comment by adamvoss** [comment]
@collin80 I won't get around to updating `m2-libraries` yet today to reflect the changes, but all the libraries from there now have their own repositories which show their relationship to upstream/origin repositories.
